.circleci/workflows.yml

```diff
@@ -486,6 +513,39 @@ commands:
             echo "Location of Google Chrome Installation: `which google-chrome-<<parameters.channel>>`"
             echo "Google Chrome Version: `google-chrome-<<parameters.channel>> --version`"
 
+  # This code builds better-sqlite3 on Debian 10 (Buster). This is necessary because Debian 10 has the oldest glibc version (2.28) that we support.
+  #
+  # Since this is running Docker remote, we need to copy the project into the container, and copy the built plugin out
```

Comment:
Can you add the reason why we're using docker remote instead of an executor with `node:20.15.0-buster` as its image? This wasn't in the centos block either, but I think it's an important detail to document, since it increases the complexity of this step

Reply:
updated comment: [31a3aec](https://github.com/cypress-io/cypress/pull/30390/commits/31a3aec956c538a661aa1b6a05e7d1301ac3b5bc)